Firestore/Example/Tests/Core/FSTQueryListenerTests.mm

```diff
@@ -148,9 +152,7 @@ - (void)testMutingAsyncListenerPreventsAllSubsequentEvents {
 
   // Drain queue
   XCTestExpectation *expectation = [self expectationWithDescription:@"Queue drained"];
-  [self.asyncQueue dispatchAsync:^{
-    [expectation fulfill];
-  }];
+  _executor->Execute([expectation] { [expectation fulfill]; });
```

Comment:
Optional: For this kind of trivial usage, maybe just capture `[=]` to cut down on noise?

The style guide says "Prefer explicit captures when the lambda will escape the current scope".